evil-jumper.el

```diff
@@ -247,14 +225,13 @@ Note: The value of `evil-jumper-file' must also be non-nil."
                  (remhash key evil-jumper--window-jumps)))
              evil-jumper--window-jumps)))
 
-(defun evil-jumper--init-file ()
-  (when (and (not evil-jumper--wired)
-             evil-jumper-file)
-    (evil-jumper--read-file)
+(defun evil-jumper--init-savehist ()
+  (unless evil-jumper--wired
+    (evil-jumper--set-window-jump-list evil-jumper--jump-list)
     (defadvice save-buffers-kill-emacs (before evil-jumper--save-buffers-kill-emacs activate)
-      (evil-jumper--write-file))
-    (when (> evil-jumper-auto-save-interval 0)
-      (run-with-timer evil-jumper-auto-save-interval evil-jumper-auto-save-interval #'evil-jumper--write-file))
+      (evil-jumper--savehist))
+    (push 'evil-jumper--jump-list savehist-additional-variables)
```

Comment:
unfortunately `savehist` is disabled by default, so some users may get a void-variable error.